README.md

```diff
@@ -12,6 +12,29 @@ This repo contains the smart contracts making up PrimeDAO.
 - `/contracts/seed/`- Prime Launch seed module contracts.
 - `/contracts/utils/`- utility contracts.
 
+## Enviornment setup
+
+create ```.env``` file and add following details
+
+```
+PROVIDER =
+MNEMONIC =
+```
+
+create ```config.json``` file and add following details
+
+```
+{
+    "SAFE": "gnosis safe address",
+    "ADMIN": "seed admin address",
+    "BENEFICIARY": "prime dao address"
+}
+```
+
+create an empty ```contract-addresses.json``` file
```

Comment:
why do we have to create an empty on if there is one in the repo?

Reply:
ohh yes
forgot that we ship the file with the repo :)